docs/content/citations.md

```diff
@@ -133,8 +133,11 @@ There are a few ways to reference your book's content, depending on what kind of
 content you'd like to reference. Here is a quick overview of some common options:
 
 * `{ref}` is used to reference section labels that you define or figures with a `name` value
+  * You can reference a section label through ``{ref}`label` `` or ``{ref}`some text <label>` ``
+  * To reference a section label in another document try ``{ref}`path/to/document/label` ``
 * `{numref}` is used to provide *numbered* references to figures
 * `{doc}` is used to reference other files in your book
+  * Documents can be referenced through ``{doc}`path/to/document` `` or ``{doc}`some text <path/to/document>` ``
```

Comment:
I wonder if both the sentence here and above could be in an extra `{tip}` block underneath this list, rather than in a nested sub-list. WDYT?

Reply:
Nice suggestion! Don't know why I did not think of this. I'll make the changes and ping you when ready for a second review.

Reply:
This has also been addressed in 413e0a9